README.md

```diff
@@ -1,113 +1,133 @@
-3. [optional] `supported_platforms`: if this file is NOT presented, it means this test case will be running in all the platforms (EC2, ECS, EKS). If this file is presented, only the platform defined inside will be applied to the test case. [For example](https://github.com/aws-observability/aws-otel-collector/blob/main/e2etest/testcases/ecsmetrics/supported_platforms). 
-4. [optional] `docker_compose.tpl`, which is used to launch sample app container in ec2 test.
-5. [optional] `ecs_taskdef.tpl`, which is used to launch ecs task in ecs test.
-6. [optional] `eks_pod_config.tpl`, which is used to launch eks pod in eks test.
+#### What does the test do?
 
-all the default files can be found [here] (https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/templates/defaults)
+this test
 
-If you find the current test case option can not fulfill your testing requirement, feel free to open an issue here so we can discuss together.
+1. Builds collector image from the directory ../aws-otel-collector
+2. Runs the collector, sample app and mock server in docker.
+3. Validates if the mock server receives data from collector.
 
-### 1.2 Open a PR to [AWS Otel Collector](https://github.com/aws-observability/aws-otel-collector) to build the new component
+## 2. How to add a new test case?
 
-You need to create a PR in [AWS Otel Collector](https://github.com/aws-observability/aws-otel-collector), "link" the testcase, and get it approved and merged. Then we will schedule to release a new version of collector with the new component.
+### 2.1 Define test case
 
-three files you need to focus on, 
+We define all the test cases under [the testcase directory](https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/testcases), and each sub folder will be treated as a test case. 
 
-* [go module to add your component](https://github.com/aws-observability/aws-otel-collector/blob/main/go.mod)
-* [enable your component](https://github.com/aws-observability/aws-otel-collector/blob/main/pkg/defaultcomponents/defaults.go).
-* [link your testcase](https://github.com/aws-observability/aws-otel-collector/blob/main/e2etest/testcases.json)
+You will need to create a sub folder under [the testcase directory](https://github.com/aws-observability/aws-otel-test-framework/tree/terraform/terraform/testcases), and place your configuration into it. Typically, 
+you will need to place files as following(please use the same filename as below):
 
-## 2. Run and debug your testcase
+1. `otconfig.tpl`: which contain the new component, will be used as the config in all types of the e2etest. 
+2. [optional] `parameters.tfvars`: you can run your test case locally with adding `-var-file=../testcases/yourtestcase/parameters.tfvars` to the terraform command. By using this file, you can override the default configuration in the testing framework. [The parameters you can override](terraform/README.md).
```

Comment:
I think we should make `parameters.tfvars` file as optional. Should we require each testcase need to be able to run locally as mandatory requirement?